### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by javlonbek97** [NVIDIA/flownet2-pytorch#288]
cuda 10.1
torch 1.5.1
python 3.7

python main.py --inference --model FlowNet2 --save_flow --save ./output --inference_dataset ImagesFromFolder --inference_dataset_root ./data/frames_seq/0_292_0_020501779/0 --resume ./models/FlowNet2_checkpoint.pth.tar
Parsing Arguments
  [0.003s] batch_size: 8
  [0.003s] crop_size: [256, 256]
  [0.003s] fp16: False
  [0.003s] fp16_scale: 1024.0
  [0.003s] gradient_clip: None
  [0.003s] inference: True
  [0.004s] inference_batch_size: 1
  [0.004s] inference_dataset: ImagesFromFolder
  [0.004s] inference_dataset_iext: jpg
  [0.004s] inference_dataset_replicates: 1
  [0.004s] inference_dataset_root: ./data/frames_seq/0_292_0_020501779/0
  [0.004s] inference_n_batches: -1
  [0.004s] inference_size: [-1, -1]
  [0.004s] inference_visualize: False
  [0.004s] log_frequency: 1
  [0.004s] loss: L1Loss
  [0.004s] model: FlowNet2
  [0.004s] model_batchNorm: False
  [0.004s] model_div_flow: 20.0
  [0.004s] name: run
  [0.004s] no_cuda: False
  [0.004s] number_gpus: 2
  [0.004s] number_workers: 8
  [0.004s] optimizer: Adam
  [0.004s] optimizer_amsgrad: False
  [0.004s] optimizer_betas: (0.9, 0.999)
  [0.004s] optimizer_eps: 1e-08
  [0.004s] optimizer_lr: 0.001
  [0.004s] optimizer_weight_decay: 0
  [0.004s] render_validation: False
  [0.004s] resume: ./models/FlowNet2_checkpoint.pth.tar
  [0.004s] rgb_max: 255.0
  [0.004s] save: ./output
  [0.004s] save_flow: True
  [0.004s] schedule_lr_fraction: 10
  [0.004s] schedule_lr_frequency: 0
  [0.004s] seed: 1
  [0.004s] skip_training: False
  [0.004s] skip_validation: False
  [0.004s] start_epoch: 1
  [0.004s] total_epochs: 10000
  [0.004s] train_n_batches: -1
  [0.004s] training_dataset: MpiSintelFinal
  [0.004s] training_dataset_replicates: 1
  [0.004s] training_dataset_root: ./MPI-Sintel/flow/training
  [0.004s] validation_dataset: MpiSintelClean
  [0.004s] validation_dataset_replicates: 1
  [0.004s] validation_dataset_root: ./MPI-Sintel/flow/training
  [0.004s] validation_frequency: 5
  [0.004s] validation_n_batches: -1
  [0.006s] Operation finished

Source Code
  Current Git Hash: b'2e9e010c98931bc7cef3eb063b195f1e0ab470ba'

Initializing Datasets
  [0.004s] Inference Dataset: ImagesFromFolder
  [0.009s] Inference Input: [3, 2, 320, 576]
  [0.041s] Inference Targets: [3, 2, 320, 576]
  [0.041s] Operation finished

Building FlowNet2 model
  [1.342s] Effective Batch Size: 16
  [1.343s] Number of parameters: 162518834
  [1.343s] Initializing CUDA
Segmentation fault
